.github/environment.yml

```diff
@@ -9,8 +9,9 @@ dependencies:
   - fftw
   - swig
   - pyyaml
-  - numpy
-  # The below is only because the currnt version of fast-pt uses deprecated scipy functions.
+  - numpy<2
```

Comment:
Have you checked that this doesn't break other CCL stuff that might have been updated to numpy 2?

Reply:
The opposite, actually. We cannot yet move to numpy 2 because many of our dependencies still don't use numpy 2. firecrown is in the same situation. This will be a tricky migration.

Reply:
You also want to add this `numpy<2` in the pyproject.toml dependencies.

Reply:
Done